src/strategies/DSL/conditions/priceCondition.ts

```diff
@@ -1,11 +1,51 @@
-import { PrimitiveArgs } from '@brinkninja/types'
-import { PriceConditionArgs } from '@brink-sdk'
+import { PrimitiveArgs, TokenArgs } from '@brinkninja/types'
+import { PriceConditionArgs, UniV3Twap, Token } from '@brink-sdk'
 
-function priceCondition (def: PriceConditionArgs): PrimitiveArgs[] {
+const TIME_INTERVAL = BigInt(1000)
+const OPERATOR_LT = 'lt';
```

Comment:
should use the types from types repo

Reply:
done. 

@mikec could you review this then? https://github.com/brinktrade/brink-types/pull/24